idx/cassandra/cassandra_test.go

```diff
@@ -285,3 +387,46 @@ func BenchmarkLoad(b *testing.B) {
 	ix.Init()
 	ix.Stop()
 }
+
+func BenchmarkIndexingWithUpdates(b *testing.B) {
```

Comment:
It seems like the goal of this test - due to it calling `insertDefs(ix, b.N)` first - is to measure how long AddOrUpdate takes if and only if no update needs to happen.  the name of this benchmark should reflect that. why is it called WithUpdates?